src/kafkerl_broker_connection.erl

```diff
@@ -185,20 +187,21 @@ handle_tcp_data(Bin, State = #state{connector = Connector, ets = EtsName,
                 end),
           case handle_errors(Errors, Messages, Name) of
```

Comment:
The expression on line 188 and column 11 is nested beyond the maximum level of 3.